aplib.net-demo/Assets/Scripts/Points/DoorPointsAdder.cs

```diff
@@ -0,0 +1,10 @@
+using Assets.Scripts.Doors;
+
+public class DoorPointsAdder : PointsAdderComponent
+{
+    private void Awake()
+    {
+        Door door = GetComponent<Door>();
+        door.DoorOpened += SendPoints;
```

Comment:
You should be subscribing to and unsubscribing from events in OnEnable() and OnDisable.